test/sampleapp/config.yaml

```diff
@@ -1,5 +1,5 @@
 languages:
-  - language: "java"
+  - language: "java-spring"
     expectedOutput: "Hello Spring Boot Sample v1!"
     preCommands:
       - exec: "curl"
```

Comment:
Sorry my bad, change 3 lines below is needed:
from: `args: "helloworld.zip -d helloworld-java_tmp"`
to:       `args: "helloworld.zip -d helloworld-java-spring_tmp"`

Reply:
done.